.travis.yml

```diff
@@ -32,7 +32,7 @@ before_install:
     chmod +x docker-compose
     sudo mv docker-compose /usr/local/bin
 
-install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q -T 2
+install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q -T 2 -pl '!presto-docs,!presto-server-rpm'
```

Comment:
Also you could remove code style checking from `install` section and other such things. This would make some tests to run disregarding code style errors. Code style is checked anyway further.